test_discovery.py

```diff
@@ -255,8 +301,8 @@ def fail_test(resp, message="", continued=False):
 class cleanText(object):
     """Mock up a module that is imported by entities so they can be imported and inspected."""
     @staticmethod
-    def text2int(wordList, spacer):
-        return wordList
+    def text2int(word_list):
+        return word_list
```

Comment:
Is this intentional? You're changing the number of arguments here

Reply:
it isn't used